src/core/annotation.js

```diff
@@ -712,3 +717,65 @@ var LinkAnnotation = (function LinkAnnotationClosure() {
 
   return LinkAnnotation;
 })();
+
+var VideoAnnotation = (function VideoAnnotationClosure() {
+  function VideoAnnotation(params) {
+    Annotation.call(this, params);
+    if (params.data) {
+      return;
+    }
+
+    function guessContentType(dict) {
+      var extToMime = {
+        'avi': 'video/avi',
+        'mov': 'video/quicktime',
+        'mpg': 'video/mpeg'
+      };
+
+      var fileName = dict.get('Movie').get('F').get('F');
+      var extName = fileName.substring(fileName.lastIndexOf('.') + 1);
+      extName = extName.toLowerCase();
+      if (!extToMime.hasOwnProperty(extName)) {
+        return null;
+      }
+      return extToMime[extName];
+    }
+
+    var dict = params.dict;
+    var data = this.data;
+    data.annotationType = AnnotationType.VIDEO;
```

Comment:
This has to be either `AnnotationType.SCREEN` or `AnnotationType.MOVIE` depending on the type of annotation. I therefore think it is best to remove this class to `MovieAnnotation` and create a new class `ScreenAnnotation` that simply inherits everything from `MovieAnnotation` and only changes this `annotationType` variable. This makes more sense since both annotation types are different and Screen annotations might have other properties than Movie annotations, so having two classes is desirable.